content/docs/hooks-overview.md

```diff
@@ -251,22 +252,22 @@ function Example() {
 }
 ```
 
-And [`useReducer`](/docs/hooks-reference.html#usereducer) lets you manage local state of complex components with a reducer:
+Z kolei [`useReducer`](/docs/hooks-reference.html#usereducer) pozwala na zarządzanie lokalnym stanem skomplikowanych komponentów przy użyciu reduktora (ang. *reducer*):
 
 ```js{2}
 function Todos() {
   const [todos, dispatch] = useReducer(todosReducer);
   // ...
 ```
 
->Detailed Explanation
+>Szczegółowe wyjaśnienie
 >
->You can learn more about all the built-in Hooks on a dedicated page: [Hooks API Reference](/docs/hooks-reference.html).
+>Więcej informacji o wszystkich wbudowanych Hookach znajdziesz w rozdziale: ["Hooki - interfejs API"](/docs/hooks-reference.html).
 
-## Next Steps {#next-steps}
+## Kolejne kroki {#next-steps}
 
-Phew, that was fast! If some things didn't quite make sense or you'd like to learn more in detail, you can read the next pages, starting with the [State Hook](/docs/hooks-state.html) documentation.
+Uff, to był wyczerpująco szybki przegląd! Jeżeli coś z tego, co przeczytałeś, nie miału sensu albo chciałbyś po prostu dowiedzieć się więcej, możesz czytać kolejne rozdziały, zaczynając od ["Hook Stanu"](/docs/hooks-state.html).
```

Comment:
```suggestion
Uff, to był wyczerpująco szybki przegląd! Jeżeli coś tu nie miało sensu lub jeśli chcesz po prostu dowiedzieć się więcej, możesz czytać kolejne rozdziały, zaczynając od ["Hook Stanu"](/docs/hooks-state.html).
```

Reply:
Tu też bardziej podobało mi się _ z tego co przeczytałeś,_

Reply:
Ok, ale w takim razie zmień to na formę bezosobową.

Reply:
Zmieniam na:

> Uff, to był wyczerpująco szybki przegląd! Jeżeli coś **z tego, co napisaliśmy,** nie miało dla ciebie sensu lub jeśli chciałbyś po prostu dowiedzieć się więcej, możesz przeczytać kolejne rozdziały, zaczynając od ["Hook Stanu"](/docs/hooks-state.html).

Reply:
> lub jeśli **chciałbyś** po prostu

Miało być bezosobowo ;-)